src/main/app/forms/models/dateOfBirth.ts

```diff
@@ -16,23 +16,27 @@ export class ValidationErrors {
 }
 
 export default class DateOfBirth implements Serializable<DateOfBirth>, CompletableTask {
+  @IsDefined ({ message: 'Choose option: Yes or Not sure' })
+  known: boolean
 
+  @ValidateIf(o => o.known === true)
```

Comment:
The new behaviour should be reflected in unit tests. Please update `app/forms/models/dateOfBirth.ts`